src/core/client/admin/components/UserMenu.tsx

```diff
@@ -0,0 +1,77 @@
+import { Localized } from "fluent-react/compat";
+import React, { StatelessComponent } from "react";
+
+import {
+  Button,
+  ButtonIcon,
+  ClickOutside,
+  Dropdown,
+  DropdownButton,
+  DropdownDivider,
+  Popover,
+} from "talk-ui/components";
+
+import styles from "./UserMenu.css";
+
+interface Props {
+  username: string;
+  onSignOut: React.EventHandler<React.MouseEvent>;
+}
+
+const UserMenu: StatelessComponent<Props> = props => (
+  <Localized id="userMenu-popover" attrs={{ description: true }}>
+    <Popover
+      id="userMenu"
+      placement="bottom-end"
+      description="A dialog of the user menu with related links and actions"
+      body={({ toggleVisibility }) => (
+        <ClickOutside onClickOutside={toggleVisibility}>
+          <Dropdown>
+            <Localized id="userMenu-viewLatestVersion">
+              <DropdownButton
+                href="https://github.com/coralproject/talk/releases"
+                target="_blank"
```

Comment:
Every `target="_blank"` should be accompanied by a `rel="noopener"`.

See https://mathiasbynens.github.io/rel-noopener/ for further details.

```suggestion
                target="_blank"
                rel="noopener"
```